hedera-node/cli-clients/src/main/java/com/hedera/services/cli/sign/AccountBalanceSigningUtils.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.cli.sign;
+
+import static com.hedera.services.cli.sign.SignUtils.TYPE_FILE_HASH;
+import static com.hedera.services.cli.sign.SignUtils.TYPE_SIGNATURE;
+import static com.hedera.services.cli.sign.SignUtils.integerToBytes;
+import static com.hedera.services.cli.sign.SignUtils.sign;
+import static com.swirlds.common.stream.LinkedObjectStreamUtilities.computeEntireHash;
+
+import com.swirlds.common.crypto.Hash;
+import com.swirlds.platform.util.BootstrapUtils;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.security.InvalidKeyException;
+import java.security.KeyPair;
+import java.security.NoSuchAlgorithmException;
+import java.security.NoSuchProviderException;
+import java.security.SignatureException;
+import java.util.Objects;
+
+/**
+ * Utility class for signing account balance files
+ */
+public class AccountBalanceSigningUtils {
+
+    /**
+     * Hidden constructor
+     */
+    private AccountBalanceSigningUtils() {}
+
+    /**
+     * Sets up the constructable registry, and configures
+     * <p>
+     * Should be called before using stream utilities
+     * <p>
+     * Do <strong>NOT</strong> call this during standard node operation, as it will interfere with static settings
+     */
+    public static void initializeSystem() {
+        BootstrapUtils.setupConstructableRegistry();
```

Comment:
Did you test whether this constructable registry setup is actually necessary? If not, this method could be removed

In any case, the commented out block below can be removed

Reply:
tested it no problem without this, removed.